app/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN mkdir -p /mattermost/data \
 
 # Configure entrypoint and command
 COPY entrypoint.sh /
+RUN chmod 750 /entrypoint.sh
```

Comment:
No, I think we should not use this instruction on the Dockerfile. I explained myself badly I guess.
Just `chmod +x entrypoint.sh` inside your Git repository and commit this change :)

Reply:
fix